src/common/Database/MySQLConnection.cpp

```diff
@@ -487,10 +487,16 @@ bool MySQLConnection::_HandleMySQLErrno(uint32 errNo)
 {
     switch (errNo)
     {
+#if defined MARIADB
+        case CR_SERVER_GONE_ERROR:
+        case CR_SERVER_LOST:
+        case CR_SERVER_LOST_EXTENDED:
+#else
         case CR_SERVER_GONE_ERROR:
         case CR_SERVER_LOST:
         case CR_INVALID_CONN_HANDLE:
         case CR_SERVER_LOST_EXTENDED:
+#endif
```

Comment:
isn't it beetter to simply put `case CR_INVALID_CONN_HANDLE:` in a NOT-MARIADB if block?

Reply:
but if i add `case CR_INVALID_CONN_HANDLE:` after else all other cases will be ignored by compiller

Reply:
miss comunication sorry

Reply:
@FrancescoBorzi done